src/main/java/org/kohsuke/github/PagedIterable.java

```diff
@@ -41,6 +44,18 @@ public PagedIterable<T> withPageSize(int size) {
         return this;
     }
 
+    /**
+     * Sets the page size. 1 by default.
+     *
+     * @param startPage
+     *            start page to set
+     * @return the paged iterable
+     */
+    public PagedIterable<T> withStartPage(int startPage) {
+        this.startPage = startPage;
```

Comment:
If this is going to be present on this class, we'll need to pass `startPage` to the `Iterator` as well.